### PR TITLE
Feature/18/generate-jwt-tokens

### LINKED ISSUE #18: [JWT] Generate tokens
## Description

* Implement manual generation of JWT secrets.
* Use secure and random methods to generate keys.
* Update environment `env` configuration.
* Ensure that the keys are unique, random, and securely generated.

## Acceptance Criteria
- [ ] Manually generate JWT secrets for access, refresh, reset, and confirmation.
- [ ] Utilize secure and random methods for key generation.
- [ ] Update the environment configuration (.env file) with the manually generated keys.
- [ ] Verify that each key is unique, random, and has sufficient entropy.

### PR DESCRIPTION
**Pull request by SeredenkoDS**
Created a script to update JWT secrets inside .env.local file. Use "npm run jwt" command in the "backend" folder to update secrets.

### CODE REVIEW

**Review by olehpona — APPROVED** [review]
Everything works perfect!

**Review by MrAK-dev — APPROVED** [review]
Looks good!

**Review by HalynaProlubshchykova — APPROVED** [review]
Everything works

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "start": "nodemon src/app.js --legacy-watch",
     "lint": "eslint . --fix",
     "prepare": "husky install",
-    "pre-commit": "lint-staged"
+    "pre-commit": "lint-staged",
+	"jwt": "node src/utils/updateEnvWithNewJWTKeys.js .env.local"
```
Comment (vlad-khrychov) [inline package.json]: Please, replace the symbol `tab` by 4 spaces. And check your VS code settings.

**Review by tufned — CHANGES_REQUESTED** [review]
Everything works as expected 👍 
But please format `updateEnvWithNewJWTKeys.js` file with _prettier_  (you can use hot keys)

Знімок екрана 2024-07-20 о 19 22 17

**Comment by SeredenkoDS** [comment]
Thank you for the comment, made the requested changes!

**Review by tufned — APPROVED** [review]
Nice!

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**